README.md

```diff
@@ -20,6 +20,20 @@ This library is part of [ACPaaS UI][acpaas-ui].
 
 ## Getting Started
 
+### Building packages
+
+The following will build `<packagename>` and it's example.
```

Comment:
This should be "its" it's comes from it is.

Reply:
Fixed.